BAZProjectC1/Models/Models.swift

```diff
@@ -0,0 +1,39 @@
+//
+//  Models.swift
+//  BAZProjectC1
+//
+//  Created by 1044336 on 15/09/22.
+//
+
+import Foundation
+
+public struct MovieApiResponseModel: Codable {
+    var page: Int?
+    var results: [MovieModel]?
+    var total_pages: Int?
+    var total_results: Int?
+    var dates: datesMovieModel?
```

Comment:
Recuerda lo comentado sobre el lowerCammelCase. Considera renombrar las propiedades.

```suggestion
    var totalPages: Int?
    var totalResults: Int?
    var dates: datesMovieModel?
```